Dockerfile.mongo

```diff
@@ -0,0 +1,6 @@
+FROM mongo:latest
+
+COPY mongodb-keyfile /data
+
+RUN chmod 600 /data/mongodb-keyfile
```

Comment:
so you dont create a user in this file, you jsut do the chmod in this file. coo.